test/tests/e2e.js

```diff
@@ -119,8 +121,7 @@ describe( 'Can Log Out', function() {
 describe( 'Can Sign up', function() {
 	this.timeout( 30000 );
 	const blogName = dataHelper.getNewBlogName();
-	const expectedBlogAddresses = dataHelper.getExpectedFreeAddresses( blogName );
-	const emailAddress = blogName + process.env.E2E_MAILOSAUR_INBOX;
```

Comment:
Is the `E2E_MAILOSAUR_INBOX` environment variable no longer necessary? If so, we should remove from the README as well as the test running script.

Reply:
Done in https://github.com/Automattic/wp-desktop/pull/785/commits/d15fdb36e6fd5a9adc5a8c3302af9db6eb7f30af

It's not necessary since we don't validate email after signing up. So we don't need a valid email address.

Reply:
We'd need to remove the reference to the mailbox env var here as well:

https://github.com/Automattic/wp-desktop/blob/569952987fec4e64fdf2d3e95bbe84327a6f8a2d/test/run/run-e2e-tests.js#L58

Reply:
Done in https://github.com/Automattic/wp-desktop/pull/785/commits/335f4c7db6090db1d5eca0286a73fb19908529b4